release.sh

```diff
@@ -16,6 +16,7 @@
 # Compiles the typescript sources to javascript and submits the files
 # to the pypi server specified as first parameter, defaults to testpypi
 # In order to run this script locally, make sure you have the following:
+# - A Python 3 environment (due to urllib issues)
```

Comment:
Is there an issue open to fix this for Py2?

Reply:
https://github.com/googledatalab/pydatalab/issues/605